### PR TITLE
Improved PV solar model

### PR DESCRIPTION
**Pull request by brookeslawski**
Improved the PV solar performance model in response to Issue #35 

- Added an example that predicts solar power of a generic 100 MW single-axis tracking PV plant and based on timeseries weather data.
- Added `solar_pysam.py`, which integrates NREL's `PySAM pvsam` to predict PV power.
- Added solar system design input file `100MW_1axis_pvsamv1.json`
- Added sample weather input file `NonAnnualSimulation-sample_data-interpolated-daytime.csv`, which was interpolated down to a time resolution of 0.5s from 1 min to match amrwind time-step

The installation instructions and the requirements will need to be updated to include PySAM v4.2.0 and restricting the Python version to 3.6.x-3.11.x.

### CODE REVIEW

File: docs/install_old.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # Installation
 Create a new conda environment for hercules:
 ```
-conda create --name hercules python
+conda create --name hercules python=3.11
```
Comment (brookeslawski) [inline docs/install_old.md:9]: I'm not sure if the python version should be specified here or if there is a way to say here that any version 3.6-3.11 is okay. I know it's specified in `setup.py`, but to my knowledge, that file isn't used at this point in the installation.

File: docs/install_old.md [hunk 2]
```diff
@@ -86,7 +96,7 @@ activate hercules`).
 
 In the first terminal, run
 ```
-helics_broker -t zmq  -f 2 --loglevel="debug"
+helics_broker -f 2 --consoleloglevel=trace --loglevel=debug --local_port=$HELICS_PORT &
```
Comment (brookeslawski) [inline docs/install_old.md:99]: The original code doesn't work for me. It throws an error (shown below).
image

File: docs/install_on_kestrel.md
Comment (brookeslawski) [inline docs/install_on_kestrel.md:1]: I added a note in line 211, but these instructions are untested. I have only run hercules locally at this point.

File: docs/install_old.md (hunk 1 above)
Comment (misi9170) [reply docs/install_old.md:9]: This is a good point. I think this is OK; in many circumstances, the user will be adding to a larger environment and they'll skip this step, but for someone creating a standalone environment this works

File: environment.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      - referencing==0.31.1
+      - requests==2.31.0
+      - rpds-py==0.13.2
+      - ruamel-yaml==0.18.5
+      - ruamel-yaml-clib==0.2.8
+      - scipy==1.11.4
+      - strip-hints==0.1.10
+      - tzdata==2023.3
+      - urllib3==2.1.0
+prefix: /Users/bstanisl/anaconda3/envs/hercules-pysam
```
Comment (misi9170) [inline environment.yml:95]: Is this file supposed to be included in the hercules repo?

**Review by misi9170 — APPROVED** [review]
All this looks good to me, and I've pulled and run the examples. Just double check whether environment.yml was intended to be included?

**Review by genevievestarke — APPROVED** [review]
Looks good!

**Pull request merged**